aodncore/pipeline/steps/resolve.py

```diff
@@ -52,17 +52,17 @@ def get_resolve_runner(input_file, output_dir, config, logger, resolve_params=No
     """
     _, file_extension = os.path.splitext(input_file)
```

Comment:
Why not just `file_type = FileType.get_type_from_name(input_file)`?

Reply:
Yep, definitely should, changed. 

I think the extension based thing was from before the FileType enum evolved into it's current form, but now it's just redundant.

Reply:
Cool, though I thought you could still store the result in a variable to avoid calling `FileType.get_type_from_name(input_file)` repeatedly...